test/cartesian_planar_graph_test.rb

```diff
@@ -0,0 +1,442 @@
+# frozen_string_literal: true
+
+# -----------------------------------------------------------------------------
+#
+# Tests for the planar graph and related classes
+#
+# -----------------------------------------------------------------------------
+
+require "test_helper"
```

Comment:
```suggestion
require_relative "test_helper"
```

This makes it so that we can `ruby thefile.rb` without need of any flag